packages/test/test-service-load/src/loadTestDataStore.ts

```diff
@@ -593,11 +592,6 @@ class LoadTestDataStore extends DataObject implements ILoadTest {
 		const opsSendType = config.testConfig.opsSendType ?? "staggeredReadWrite";
 		const opsPerCycle = (config.testConfig.opRatePerMin * cycleMs) / 60000;
 		const opsGapMs = cycleMs / opsPerCycle;
-		const futureOpPeriod =
-			config.testConfig.futureOpRatePerMin === undefined ||
```

Comment:
Two other files to update related to this:  testConfigFile.ts and testConfig.json